src/order-requests/order-requests.service.ts

```
@@ -110,35 +116,53 @@ export class OrderRequestsService {
 
     if (!orderRequest) {
       throw new NotFoundException('해당 주문 요청을 찾을 수 없습니다.');
     }
-
+  
+    // 🔹 요청한 사람(requesterId)과 처리한 사람(resolverId)의 이름 조회
+    const requester = await this.prisma.user.findUnique({
+      where: { id: orderRequest.requesterId },
+      select: { name: true },
+    });
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)